recipes/massren/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "massren" %}
+{% set version = "1.5.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/laurent22/massren/archive/v{{ version }}.tar.gz
+  sha256: 49758b477a205f3fbf5bbe72c2575fff8b5536f8c6b45f8f6bd2fdde023ce874
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('go-nocgo') }}
+    - go-licenses
+    - sqlite
```

Comment:
This looks like it should be in host.

Reply:
I have fixed this in the most recent commit.